build/devenv/tests/e2e/smoke_test.go

```diff
@@ -108,27 +117,20 @@ func TestE2ESmoke(t *testing.T) {
 		chains[2].Details.ChainSelector
 
 	t.Run("extra args v2", func(t *testing.T) {
-		tcs := []struct {
-			name                     string
-			fromSelector             uint64
-			toSelector               uint64
-			receiver                 protocol.UnknownAddress
-			expectFail               bool
-			numExpectedVerifications int
-		}{
+		tcs := []v2TestCase{
 			{
 				name:                     "src->dst msg execution eoa receiver",
 				fromSelector:             sel0,
 				toSelector:               sel1,
-				receiver:                 mustGetEOAReceiverAddress(t, chains[1], sel1),
+				receiver:                 mustGetEOAReceiverAddress(t, chainMap[sel1]),
```

Comment:
Switching to chainMap to avoid coupling to the chains[0] referring to sel0, etc.